### PR TITLE
Refactor dependencies

### PR DESCRIPTION
**Pull request by zoepage**

### CODE REVIEW

**Comment by zoepage** [comment]
FYI: semantic-release@latest build fails with travis.

File: package.json [hunk 1]
```diff
@@ -4,10 +4,9 @@
   "description": "initials for names",
   "main": "initials.js",
   "scripts": {
-    "jshint": "jshint -c .jshintrc *.js test/*.js",
-    "test": "npm run jshint && testmate test/*.js",
-    "test:node": "testmate --client node test/*.js",
-    "test:browser": "testmate --client selenium:firefox test/*.js",
+    "test": "standard initials.js test/initials-test.js && npm run -s test:node | tap-spec",
```
Comment (gr2m) [inline package.json]: can you move "standard" into "pretest"? And if you want to ignore files / folders from being linted by standard, check https://www.npmjs.com/package/standard#how-do-i-ignore-files

**Comment by gr2m** [comment]
Looking great, just see my comment above. Can you clean up the commits?

**Comment by gr2m** [comment]
That's looking great! Only problem I see is that now the [index.html](index.html) can't load it's dependencies anymore, and gh-pages will be broken.

Ideally, we would develop on master branch, and have a deploy task that pushes what ever files we want to gh-pages, similar to how we deploy the doxx files in pouchdb-hoodie-api. But we can do that next, and for now load the files from `bower_components/` from the web, and load initials.js from wzrd.in (https://wzrd.in/standalone/initials)

What do you think?

**Comment by gr2m** [comment]
done :)

**Pull request merged**